### ISSUE TITLE
about conv layer number

### ISSUE THREAD

**Issue by jastfkjg** [pl8787/MatchPyramid-TensorFlow#14]
Hello!  

In the paper, models use two conv layers and two max-pooling layers. 

But in your code, there is only one. Is one layer comes out with a better result ?

Another small question, do we need to padding X1, X2 ? and what does `X1_len`, `X2_len`, `Y`, `F` mean in model_mp.py ?

Thanks.

**Comment by pl8787** [comment]
It is depended on your task and training dataset, if your task is hard and the dataset is large enough, you may use more conv layers.

You do not have to pad X1 and X2. X1_len and X2_len are the maximum length of X1 and X2. Y means the label and you can ignore F (which is additional features).

**Comment by jastfkjg** [comment]
OK, thanks a lot!
